Pod/Classes/ViewControllers/ARMapContainerViewController.m

```diff
@@ -0,0 +1,38 @@
+//
+//  ARMapContainerViewController.m
+//  Emission
+//
+//  Created by Luc Succes on 12/19/18.
+//
+
+#import "ARMapContainerViewController.h"
+#import "ARMapComponentViewController.h"
+#import "ARCityComponentViewController.h"
+
+#import "Emission-Swift.h"
+
+@interface ARMapContainerViewController ()
+
+@property (nonatomic, readwrite) BottomSheetViewController *bottomSheetVC;
+@property (nonatomic, readwrite) ARMapComponentViewController *mapVC;
+@property (nonatomic, readwrite) ARCityComponentViewController *cityVC;
+
+@end
+
+@implementation ARMapContainerViewController
```

Comment:
This class will also be responsible for inter City / Map view communication such as applying filters across both and changing cities / location